.gitlab-ci.yml

```diff
@@ -48,15 +48,15 @@ RPM:
           - aws/centos-stream-9-aarch64
           - aws/rhel-8.6-ga-x86_64
           - aws/rhel-8.6-ga-aarch64
+          - aws/rhel-8.7-ga-x86_64
```

Comment:
I wonder if we should not delete the 8.6 and 9.0 GA runners? We do not test on these RHEL versions in any of the pipelines listed in this file. 🤔

Reply:
we test manifests on 8.6 and 9.0, so we need to keep the builds as long as we test those...

Reply:
we should probably remove these yea :D Let's do in followup? Unless the selinux package still isn't synced.